[2022-08-06 19:37:03,808] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 19:37:03,823] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 19:37:03,824] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:03,824] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:37:03,824] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:03,843] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-10 00:00:00+00:00
[2022-08-06 19:37:03,850] {standard_task_runner.py:52} INFO - Started process 1076 to run task
[2022-08-06 19:37:03,853] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-10T00:00:00+00:00', '--job-id', '314', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpozpyy3ai', '--error-file', '/tmp/tmp82wr8lpe']
[2022-08-06 19:37:03,853] {standard_task_runner.py:80} INFO - Job 314: Subtask second_task
[2022-08-06 19:37:03,928] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:37:04,017] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-10T00:00:00+00:00
[2022-08-06 19:37:04,018] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:37:04,018] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:37:04,031] {subprocess.py:85} INFO - Output:
[2022-08-06 19:37:04,032] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:37:04,033] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:37:04,066] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211110T000000, start_date=20220806T193703, end_date=20220806T193704
[2022-08-06 19:37:04,104] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:37:04,136] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:50:56,834] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 21:50:56,856] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 21:50:56,857] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:56,857] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:50:56,857] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:56,881] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-10 00:00:00+00:00
[2022-08-06 21:50:56,890] {standard_task_runner.py:52} INFO - Started process 1269 to run task
[2022-08-06 21:50:56,900] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-10T00:00:00+00:00', '--job-id', '317', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpczdejed0', '--error-file', '/tmp/tmpg63t6_3y']
[2022-08-06 21:50:56,900] {standard_task_runner.py:80} INFO - Job 317: Subtask second_task
[2022-08-06 21:50:56,986] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:50:57,080] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-10T00:00:00+00:00
[2022-08-06 21:50:57,081] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:50:57,081] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:50:57,097] {subprocess.py:85} INFO - Output:
[2022-08-06 21:50:57,099] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:50:57,099] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:50:57,149] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211110T000000, start_date=20220806T215056, end_date=20220806T215057
[2022-08-06 21:50:57,192] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:50:57,236] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:56:23,440] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 21:56:23,455] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 21:56:23,455] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:23,455] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:56:23,455] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:23,472] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-10 00:00:00+00:00
[2022-08-06 21:56:23,478] {standard_task_runner.py:52} INFO - Started process 1068 to run task
[2022-08-06 21:56:23,481] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-10T00:00:00+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp005k05y7', '--error-file', '/tmp/tmpmpl3kyay']
[2022-08-06 21:56:23,481] {standard_task_runner.py:80} INFO - Job 316: Subtask second_task
[2022-08-06 21:56:23,551] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:56:23,656] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-10T00:00:00+00:00
[2022-08-06 21:56:23,656] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:56:23,657] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:56:23,671] {subprocess.py:85} INFO - Output:
[2022-08-06 21:56:23,673] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:56:23,673] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:56:23,717] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211110T000000, start_date=20220806T215623, end_date=20220806T215623
[2022-08-06 21:56:23,772] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:56:23,802] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:11:27,142] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 22:11:27,154] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 22:11:27,154] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:27,154] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:11:27,154] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:27,170] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-10 00:00:00+00:00
[2022-08-06 22:11:27,177] {standard_task_runner.py:52} INFO - Started process 1082 to run task
[2022-08-06 22:11:27,180] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-10T00:00:00+00:00', '--job-id', '319', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp4m2q9p5b', '--error-file', '/tmp/tmph3opavke']
[2022-08-06 22:11:27,180] {standard_task_runner.py:80} INFO - Job 319: Subtask second_task
[2022-08-06 22:11:27,266] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:11:27,398] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-10T00:00:00+00:00
[2022-08-06 22:11:27,400] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:11:27,401] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:11:27,419] {subprocess.py:85} INFO - Output:
[2022-08-06 22:11:27,422] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:11:27,422] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:11:27,533] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211110T000000, start_date=20220806T221127, end_date=20220806T221127
[2022-08-06 22:11:27,560] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:11:27,613] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:19:10,709] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 22:19:10,723] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 22:19:10,723] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:19:10,723] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:19:10,723] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:19:10,740] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-10 00:00:00+00:00
[2022-08-06 22:19:10,747] {standard_task_runner.py:52} INFO - Started process 1085 to run task
[2022-08-06 22:19:10,751] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-10T00:00:00+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpgthx_u6e', '--error-file', '/tmp/tmp9bq_a9i_']
[2022-08-06 22:19:10,752] {standard_task_runner.py:80} INFO - Job 316: Subtask second_task
[2022-08-06 22:19:10,839] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [running]> on host ed53bde1c44a
[2022-08-06 22:19:10,940] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-10T00:00:00+00:00
[2022-08-06 22:19:10,941] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:19:10,942] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:19:10,956] {subprocess.py:85} INFO - Output:
[2022-08-06 22:19:10,957] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:19:10,957] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:19:11,014] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211110T000000, start_date=20220806T221910, end_date=20220806T221911
[2022-08-06 22:19:11,042] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:19:11,089] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:22:47,265] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 22:22:47,284] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [queued]>
[2022-08-06 22:22:47,284] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:47,284] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:22:47,284] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:47,304] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-10 00:00:00+00:00
[2022-08-06 22:22:47,311] {standard_task_runner.py:52} INFO - Started process 1115 to run task
[2022-08-06 22:22:47,313] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-10T00:00:00+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp031laao3', '--error-file', '/tmp/tmpf67f4is5']
[2022-08-06 22:22:47,314] {standard_task_runner.py:80} INFO - Job 316: Subtask second_task
[2022-08-06 22:22:47,400] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-10T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:22:47,519] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-10T00:00:00+00:00
[2022-08-06 22:22:47,520] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:22:47,521] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:22:47,540] {subprocess.py:85} INFO - Output:
[2022-08-06 22:22:47,543] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:22:47,543] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:22:47,587] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211110T000000, start_date=20220806T222247, end_date=20220806T222247
[2022-08-06 22:22:47,645] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:22:47,691] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
